README.md

```diff
@@ -2,12 +2,16 @@
 backend for the task mgmt tool
 
 Description of the project:
-**hello**
 
-halo
-bonjour
-ni hao
 
-Authors:
-Zee
+
+
+
+Dependency:
+```python
+pip install -r requirements.txt
```

Comment:
@ahamed-sha please run this command from the Readme.md file to install the dependencies for this project. I have added sql-lite database as dependency here.